src/client.js

```diff
@@ -360,6 +360,21 @@ MatrixClient.prototype.getStoredDevicesForUser = function(userId) {
 };
 
 /**
+ * Get the stored device key for a user id and device id
+ *
+ * @param {string} userId the user to list keys for.
+ * @param {string} deviceId unique identifier for the device
+ *
+ * @return {module:crypto-deviceinfo} device or nothing
```

Comment:
since its nullable it should be
`@return {?module:crypto-deviceinfo} device or null`